libraries/AP_BattMonitor/AP_BattMonitor_SMBus_SUI.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
```

Comment:
No reason to include AP_Param.h here that I can see? I know it's coming in transitively via other headers, but we don't actually need it, no reason to make it harder to fix the other headers then it already is...

Reply:
fixed